.gitignore

```diff
@@ -1,26 +1,2 @@
-
-# Binaries for programs and plugins
```

Comment:
Why all the content of the file was removed? It was originally auto generated and contains patterns that is useful to ignore.

Reply:
We don't need to ignore what doesn't exists
You add only what you plan to ignore in the PR that creates new stuff that need to be ignored
This is one of the drawbacks of using generator
See please all the other repos, the git ignore should be minimal
As long as git status is clean, the less the merrier